src/app/containers/Article/index.jsx

```diff
@@ -35,6 +35,16 @@ class ArticleContainer extends Component {
 
   render() {
     const { data } = this.props;
+
+    /*
+      This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
+      After.JS gives no explicit loading state, so we just have to check for a lack of data.
+      After.JS should handle async data fetching in a more informative way, which will be become an issue for error handling.
+    */
+    if (!data) {
+      return 'Loading...';
```

Comment:
This should be a separate PR, alongside the change here https://github.com/bbc/simorgh/pull/538/files#diff-b83d3a0f6ef43a3dcaea2ca5421e2743L48

Reply:
It could very well be in a PR, but that PR would have to be merged before implementing React Router Links. I'm happy to do that work, but we should also consider whether that work would be valuable when we are looking to get this work through fairly quickly.

Reply:
This feels a fairly large PR. I would break it down into separate concerns.

Reply:
@jp-bbc can you give points where it should be split into smaller tasks? Please also see a justification for not doing that in the comment above.